Assets/Scripts/Combat/Abilities/Ability.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
Remove Unnecessary packages. They throw warnings sometimes

Reply:
Done! Did it for all other files.